pkg/apm/labels.go

```diff
@@ -39,7 +43,9 @@ func (apm *APM) GetLabel(key string) (*Label, error) {
 		}
 	}
 
-	return nil, fmt.Errorf("no label found with key %s", key)
+	return nil, &errors.ErrorNotFound{
+		Message: fmt.Sprintf("no label found with key %s", key),
```

Comment:
This is happening throughout the PR, feels like a good place to make a `New` method, or something similar to `fmt.Errorf`

```
  return nil, errors.NewNotFoundf("no condition found for policy %d and condition ID %d", policyID, pluginID)
```

Reply:
👍